client/src/components/mainpage/othercards.js

```diff
@@ -6,7 +6,7 @@ import "./othercards.css"
 function OtherCards(props) {
 
     return (
-        <div className="col s12 m6 l4 xl3">
```

Comment:
We need to keep this!

Reply:
How can I fix this? close commit and add the code line and re commit?? I am not sure how to.

Reply:
I added "  <div className="col s12 m6 l4 xl3">   " back in.  Hopefully this will work.